atomic_reactor/util.py

```diff
@@ -785,3 +787,71 @@ def get_config_from_registry(image, registry, digest, insecure=False,
     logger.debug('Image %s:%s has config:\n%s', context, tag, blob_config)
 
     return blob_config
+
+def df_parser(df_path, workflow=None, cache_content=False, env_replace=True, parent_env=None):
```

Comment:
`DockerfileParser` is a library that takes into account different use cases. 
We only use a subset of it. Since this is a util function in `atomic-reactor`, it can be made more specific which would lessen complexity.

To make this as easy to use as possible, I think we should only have to pass workflow object.